src/_P021_Level.ino

```diff
@@ -244,19 +244,20 @@ boolean Plugin_021(uint8_t function, struct EventStruct *event, String& string)
 
     case PLUGIN_WEBFORM_LOAD:
     {
-      const bool extFunc = bitRead(P021_FLAGS, P021_EXT_FUNCT); // Extended functionality selected
+      const uint16_t flags   = P021_FLAGS;                     // shortcut to read existing configuration flags
```

Comment:
This indeed does save quite a bit in build size, as the macro would otherwise be expanded quite a lot of times.

Reply:
Yeah, I abuse one of the PCONFIG parameters to store a bunch of booleans, By the way, I made a slip of the pen at the webform_save. Forgot to set one flag using the new variable. Will upload the change soon.